c++/src/capnp/layout.c++

```diff
@@ -1609,31 +1636,61 @@ struct WireHelpers {
       return { segment, ptr };
     } else {
       // List of structs.
+      WordCount declDataSize = roundBitsUpToWords(value.structDataSize);
```

Comment:
In the INLINE_COMPOSITE case, the tag word has the layout of a struct pointer. Within a struct pointer, the data section size is a count of words, so there's no way it would not be evenly divisible by words here, right?